### PR TITLE
{Release} Upgrade to Azure CLI 2.6.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,118 @@
...
+**AKS**
+
+* Update uptime-sla command help context (#13300)
+* Remove range check for updating min count for autoscaler (#13215)
+* Fix that cli doe not fail when user only specifies Windows password (#13418)
+
+**AMS**
+
+* Change: az ams transform create: Added ability to create a transform with a FaceDetector preset (#13260)
+* CHange: az ams content-key-policy create : Added ability to create a FairPlay content key policy with an offline rental configuration (#13260)
```
Comment (fengzhou-msft) [inline src/azure-cli/HISTORY.rst:29]:
```suggestion
* `az ams transform create`: Add ability to create a transform with a FaceDetector preset (#13260)
* `az ams content-key-policy create` : Add ability to create a FairPlay content key policy with an offline rental configuration (#13260)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,118 @@
...
+**ACR**
+
+* Add default timeout of 5 minutes for any requests to ACR (#13349)
+* Support disable public network access (#13347)
+* `az acr token create`: expose --days argument (#13392)
+* `az acr import`: accept --source argument values which contain login in server name through client end correction (#13392)
+
+**ACS**
+
+* bug fix: remove fields cleanup for fields that no longer exist (#13315)
```
Comment (yungezz) [inline src/azure-cli/HISTORY.rst:18]:
```suggestion
* Bug fix: remove fields cleanup for fields that no longer exist (#13315)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,118 @@
...
+
+**AKS**
+
+* Update uptime-sla command help context (#13300)
+* Remove range check for updating min count for autoscaler (#13215)
+* Fix that cli doe not fail when user only specifies Windows password (#13418)
+
+**AMS**
+
+* Change: az ams transform create: Added ability to create a transform with a FaceDetector preset (#13260)
```
Comment (yungezz) [inline src/azure-cli/HISTORY.rst:28]:
```suggestion
* Change: az ams transform create: Add ability to create a transform with a FaceDetector preset (#13260)
```

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (yungezz) [inline src/azure-cli/HISTORY.rst:29]:
```suggestion
* Change: az ams content-key-policy create : Add ability to create a FairPlay content key policy with an offline rental configuration (#13260)
```

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:18]: This a bad description/history. 

> Fix #xxx: `az xxx`: xxx

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,118 @@
...
+* CHange: az ams content-key-policy create : Added ability to create a FairPlay content key policy with an offline rental configuration (#13260)
+
+**AppConfig**
+
+* Bug fix for list key values with fields (#13326)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:33]: This a bad description/history.

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,118 @@
...
+
+**CI**
+
+* Support flake8 3.8.0 (#13454)
+
+**Compute**
+
+* New command az vm auto-shutdown (#13199)
+* `az vm list-skus`: Update --zone behavior, return all type skus now (#13470)
+
```
Comment (arrownj) [inline src/azure-cli/HISTORY.rst:71]:
```suggestion

2.6.0
++++++
* Update local context on/off status to global user level (#13277)

```

**Pull request merged**